docs/envvars.md

```diff
@@ -128,12 +128,12 @@ When doing local development, it's recommended to set this to
 
 The base URL used in the Interactive Example iframes.
 
-### `BUILD_LOG_MACROS_USED`
+### `BUILD_LOG_MACROS_USED_LOGFILE`
```

Comment:
@peterbe I was thinking that `BUILD_MACROS_USED_LOGFILE` might be less verbose and better? What do you think?

Reply:
Updated. r?